### DISCUSSION TITLE
DatePicker calendar expand icon dark mode color

### DISCUSSION THREAD

**Discussion by vankhoawin** [twilio-labs/paste#1609] [General] [answered]
Hello team, very minor issue: in dark mode, the calendar expand icon doesn't have the same color as the text or the caret icon. DatePicker looks great otherwise!

https://paste.twilio.design/components/date-picker/

![image](https://user-images.githubusercontent.com/6936736/123673070-e3df8200-d7f4-11eb-92b2-b763890e2739.png)

**Comment by richbachman** [comment, marked as answer]
Hi @vnguyen94. Thanks for posting this. We're aware of this issue. Unfortunately that icon is browser generated/controlled so we're kind of stuck with it. If you have any ideas on how to work around that, let us know. Otherwise we'll post here if/when we ever work out a workaround.

**Reply by vankhoawin** [reply]
ahh gotcha, definitely harder to deal with if it's browser dependent. thanks for the quick response!
